app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ProjectsController < ActionController::Base
+  # def index
+
+  # end
+
+  def show
+
+    # require 'pry'; binding.pry
+    @project = Project.find(params[:id])
+    @theme = Challenge.find(@project.challenge_id)
```

Comment:
All you need for this controller action is actually line 9 (@project). We can reach through this object in the view to get what we need
We can get the theme by doing `@project.challenge.theme`
We can get the contestant_projects associated by doing `@project.contestant_projects`